serverless-functions/src/functions/common/twilio-wrappers/sync.private.js

```diff
@@ -4,6 +4,123 @@ const retryHandler = require(Runtime.getFunctions()[
   "common/twilio-wrappers/retry-handler"
 ].path).retryHandler;
 
+/**
+ * @param {object} parameters the parameters for the function
+ * @param {number} parameters.attempts the number of retry attempts performed
+ * @param {object} parameters.context the context from calling lambda function
+ * @param {string} parameters.mapSid the SID of the Sync Map
+ * @param {string} parameters.key the key of the Sync Map item
+ * @returns {object} A new Sync Map Item
```

Comment:
Does this actually return a map item? I'd generally expect a delete to just return success/status, but the documentation doesn't indicate anything is returned.